packages/experimental/Switch/src/Switch.tsx

```diff
@@ -29,19 +29,34 @@ export const switchLookup = (layer: string, state: SwitchState, userProps: Switc
   );
 };
 
+const isMobile = Platform.OS === 'android' || Platform.OS == 'ios';
+
 export const Switch = compose<SwitchType>({
   displayName: switchName,
   ...stylingSettings,
   slots: {
     root: Pressable,
     label: Text,
-    track: View,
-    thumb: View,
+    track: Animated.View, // Conversion from View to Animated.View for Animated API to work
+    thumb: Animated.View,
     toggleContainer: View,
     onOffText: Text,
   },
   useRender: (userProps: SwitchProps, useSlots: UseSlots<SwitchType>) => {
-    const switchInfo = useSwitch(userProps);
+    const switchOnSlot = useSlots(userProps, (layer) => switchLookup(layer, { toggled: true }, {}));
+    const switchOffSlot = useSlots(userProps, (layer) => switchLookup(layer, { toggled: false }, {}));
+
+    // For Mobile platform we are passing extra data to useSwitch for Animated API
+    const switchInfo = useSwitch(
+      userProps,
+      isMobile && {
```

Comment:
I think using `Animated` for the animation would also benefit Windows and macOS, so doesn't have to be constrained to mobile. More, "not-win32".

Reply:
@Saadnajmi , agree on that point. `Animated` work cross-plat but I would like to take it forward with mobile first in its first implementation. 
There're couple of things are need from the design and for mobile it has clear definition, as animating using Animated is doing using `transform` not given the react native to handle such as Layout Animation does.  

There will few tokens changes/value might require on win32 as they're used in calculations for x-axis transformation according to padding between track * knob and width of the track and other things such as speed & duration of the time, hence it would be great if we could do that in a separate PR. For iOS i will test and add screen recordings.

Reply:
Fair enough if you want to keep it limited to the platforms you've tested.